cmta/src/main/scala/cmt/admin/command/Studentify.scala

```diff
@@ -0,0 +1,148 @@
+              options.studentifyBaseDirectory.value.getPath)}")
+
+          mainRepoName = mainRepository.value.getName
+          tmpFolder = sbtio.createTemporaryDirectory
+          cleanedMainRepo = tmpFolder / mainRepoName
+          studentifiedRootFolder = options.studentifyBaseDirectory.value / mainRepoName
+          solutionsFolder = studentifiedRootFolder / config.studentifiedRepoSolutionsFolder
+
+          _ = checkpreExistingAndCreateArtifactRepo(
+            options.studentifyBaseDirectory.value,
+            studentifiedRootFolder,
+            options.forceDelete.value)
+
+          _ = sbtio.createDirectory(options.studentifyBaseDirectory.value / config.studentifiedRepoSolutionsFolder)
+
+          _ <- copyCleanViaGit(mainRepository.value, tmpFolder, mainRepoName)
+
+          ExercisesMetadata(prefix, exercises, exerciseNumbers) <- getExerciseMetadata(mainRepository.value)(config)
+
+          _ = buildStudentifiedRepository(
+            cleanedMainRepo,
+            exercises,
+            studentifiedRootFolder,
+            solutionsFolder,
+            config,
+            options.initGit,
+            tmpFolder)
+
+          successMessage <- Right(exercises.mkString("Processed exercises:\n  ", "\n  ", "\n"))
+
+        } yield successMessage
+      end execute
+  end given
+
+  private object StudentifyHelpers:
+    def buildStudentifiedRepository(
+        cleanedMainRepo: File,
+        exercises: Vector[String],
+        studentifiedRootFolder: File,
+        solutionsFolder: File,
+        config: CMTaConfig,
+        initializeAsGitRepo: InitializeGitRepo,
+        tmpFolder: File): Either[CmtError, String] =
+
+      addFirstExercise(cleanedMainRepo, exercises.head, studentifiedRootFolder)(config)
+
+      writeTestReadmeCodeMetadata(cleanedMainRepo, exercises, studentifiedRootFolder, config)
+
+      hideExercises(cleanedMainRepo, solutionsFolder, exercises)(config)
+
+      writeStudentifiedCMTConfig(studentifiedRootFolder / config.cmtStudentifiedConfigFile, exercises)(config)
+      writeStudentifiedCMTBookmark(studentifiedRootFolder / config.studentifiedRepoBookmarkFile, exercises.head)
+
+      val successMessage = exercises.mkString("Processed exercises:\n  ", "\n  ", "\n")
+      if initializeAsGitRepo.value then
+        val dotIgnoreFile = cleanedMainRepo / ".gitignore"
+        if dotIgnoreFile.exists then sbtio.copyFile(dotIgnoreFile, studentifiedRootFolder / ".gitignore")
+        for {
+          _ <- initializeGitRepo(studentifiedRootFolder)
+          _ <- commitToGit("Initial commit", studentifiedRootFolder)
+          _ = sbtio.delete(tmpFolder)
+          result <- Right(successMessage)
+        } yield result
+      else
+        sbtio.delete(tmpFolder)
+        Right(successMessage)
+    end buildStudentifiedRepository
+  end StudentifyHelpers
+
+  val command = new CmtCommand[Studentify.Options] {
+
+    def run(options: Studentify.Options, args: RemainingArgs): Unit =
```

Comment:
Wondering what the `args` parameter is doing as it doesn't seem to be utilised in the `run` method.